### PR TITLE
Add Dynamic data on events section cards

### LINKED ISSUE #100: 🐛 Bug Report: Add Dynamic data on events section cards
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently data on events section  i.e. in cards are hard coded .

Dynamic data

### Expected Behavior

we want data in cards to  be dynamic (data will come from server). As of now you can  store data in JSON files , JS file instead of Server l. **Main point is data should be dynamic.**

### Steps To Reproduce

You can use JSON Server and Axios or any API of your choice to make it work.

### Anything else?

_No response_

### What browsers are you seeing the problem on?

_No response_

### PR DESCRIPTION
**Pull request by aryanda1**
**Notes for Reviewers**

This PR fixes #100 

Describe the changes you've made:
Added a all_events javascript array, which contains all the events, and than use it to display the appropriate events

Screenshots for the change:

### CODE REVIEW

**Comment by kamalchakma1** [comment]
@aryanda1 Great work with explanation within comments.

**Review by kamalchakma1 — DISMISSED** [review]

**Review by ArunPatel02 — DISMISSED** [review]

File: src/Pages/Event/Event.js [hunk 1]
```diff
@@ -2,49 +2,58 @@ import React, { useEffect } from 'react'
...
+    // ])
+    // const allEvents = upcomingdata.concat(pastData) 
+    // /* allEvents is an array which consists of both upcoming and past events */
+    // const [allData, setAllData] = useState(allEvents)
+
+    // useEffect(() => {
+    //     setAllData(allData)
+    // }, [allData])
+
+    // useEffect(() => {
```
Comment (Yashsharma1911) [inline src/Pages/Event/Event.js:54]: @aryanda1 if we are not using these line of code can we delete them or there is any specific reason for keeping them

**Comment by Yashsharma1911** [comment]
some good work by @aryanda1 👏

File: src/Pages/Event/Event.js (hunk 1 above)
Comment (aryanda1) [reply src/Pages/Event/Event.js:54]: Well they were in the original code. And i saw in some other PR code, they commented out the old stuff. So i wasnt sure what to do. Should i delete it?

Comment (Yashsharma1911) [reply src/Pages/Event/Event.js:54]: Then please delete them and then I think we will be good to go 😎

Comment (aryanda1) [reply src/Pages/Event/Event.js]: Done check

**Pull request merged**